app/migration/Main.hs

```diff
@@ -92,13 +96,33 @@ main = do
           (UniqueUserName userName)
           (User userName hash' Nothing False False False)
           [ UserPasswordHash P.=. hash'
-          , UserApiToken P.=. Nothing
```

Comment:
If a totally new user is created with the intention of replacing the old one, the old user should be deleted first with `deleteuser`. If the intention is to just delete the API key, `deleteapikey` should be used. Either way, I think `createuser` shouldn't remove the API token upon subsequent calls. Though I don't have strong opinions on this and can be convinced otherwise.